src/resiliation/resiliation-reason.component.js

```diff
@@ -49,7 +49,7 @@ export default {
       self.panel = false;
       self.onChange({
         ELEMENT: self.tucResiliationReason,
-        SURVEY: self.tucResiliationReasonModel,
+        SURVEY: self.resiliationReasonModel,
```

Comment:
I think there is a problem here : https://github.com/ovh-ux/ovh-manager-telecom/blob/213b4783df/client/app/telecom/pack/xdsl/resiliation/pack-xdsl-resiliation.html#L70-L75

It should be 

```html 
<tuc-resiliation-reason
    data-ng-show="PackXdslResiliation.resiliationTerms.status"
    data-tuc-resiliation-reason-filter="{{PackXdslResiliation.resiliationTerms.data.resiliationReasons.join(',')}}"
    data-tuc-resiliation-reason="{id: PackXdslResiliation.resiliationTerms.serviceName, name:PackXdslResiliation.resiliationTerms.serviceName, type:'service', typeName:PackXdslResiliation.resiliationTerms.typeName}"
    on-change="PackXdslResiliation.resiliateService(ELEMENT, SURVEY, ACCEPT)">
</tuc-resiliation-reason>
```

(wrong `data-tuc-resiliation-reason-filter` binding).

We have another problem in this same code, there is no `tucResiliationReasonModel` defined https://github.com/ovh-ux/ng-ovh-telecom-universe-components/blob/f09253603bd2cb81d2219c8eecf57cace0477cdb/src/resiliation/resiliation-reason.component.js#L9-L14

so the `canResiliate` method will always return `false` 
https://github.com/ovh-ux/ng-ovh-telecom-universe-components/blob/f09253603bd2cb81d2219c8eecf57cace0477cdb/src/resiliation/resiliation-reason.component.js#L38-L46

Reply:
Even better find :D

Reply:
Thanks @FredericEspiau !

Reply:
@cbourgois no it won't, I made some tests on it.

Reply:
@cbourgois I made the fix about data-tuc-resiliation-reason on telecom manager

Reply:
@Steffy29 Shouldn't be `tucResiliationReasonModel` instead?

Reply:
No I made some tests yesterday with the older version with `self.tucResiliationReasonModel` and the `SURVEY` wasn't set.